docs/adr/adr-0022-new-icon-system.md

```diff
@@ -0,0 +1,232 @@
+* A `<title>` element which value matches the value of the `accessible-name` prop.
+
+The addition of the `<title>` element is not necessary but this is the native HTML way for informative `<svg>` elements so we decided to keep it.
+
+Example of an informative `<cc-icon>`:
+```html
+<cc-icon accessible-name="Caution">
+  <svg role="img" aria-label="Caution">
+    <title>Caution</title>
+    ...
+  </svg>
+<cc-icon>
+```
+
+#### Deciding on prop names
+
+We discussed several options about the component API and how developers should use it:
+
+##### Using ARIA attributes directly
+
+This option was only possible if we used the `<cc-icon>` element itself (see "Using the `<svg>` element itself" above).
+
+Developers would directly add `aria-hidden=true` or `role="img"` + `aria-label` to the `<cc-icon>` element itself.
+
+We did not choose to go this way because:
+
+* There was too much room for mistakes. Developers could add the `role="img"` attribute and forget to add the `aria-label` attribute.
+* Relying on the `<cc-icon>` element itself and not the `<svg>` does not work on Safari.
+
+##### Reproducing the img API
+
+We considered using an `alt` prop and attribute to mimic the `<img>` API.
+
+We did not choose to go this way because we want to avoid using existing native attributes like `title` for instance.
+
+##### Using a custom accessible-name attribute
+
+We chose to use a `accessible-name` attribute that should only be used if the icon conveys information.
+
+This simplifies the API for developers because:
+
+* In most cases, icons are decorative so there is nothing to do.
+* When the icon is informative, they only have one attribute to add.
+* We already use an `accessible-name` attribute on the `<cc-button>`. This means this attribute should be familiar for developers using our components. It is also a well-known concept for many people who have worked on accessibility before.
+
+## Limits and prospects
+
+Where Remix Icon is a great library, it is currently on [pause](https://github.com/Remix-Design/RemixIcon/issues/232).
+
+As we strictly limited the first version of the new icon system to our use cases, there are some options worth considering for future improvements:
+- add more properties to the component for tweaking the icon rendering: `flip`, `rotation`, `badge`, etc.
+- implement an icon [provider](https://core.clarity.design/foundation/icons/) to facilitate usage outside the Clever Components scope.
+
+## Resources
+
+### In English
+
+* [a11y support - `<svg>` + `<title>` elements](https://a11ysupport.io/tests/html_svg_inline_with_title)
+* [Accessible SVGs: Perfect Patterns For Screen Reader Users by Carie Fisher on Smashing Magazine](https://www.smashingmagazine.com/2021/05/accessible-svg-patterns-comparison/)
+
+### In French
+
+* <a href="https://access42.net/svg-accessible?lang=fr" lang="fr">Comment intégrer du SVG de manière accessible ? - Access42</a>,
```

Comment:
Why not use the markdown syntax for those links?

Reply:
I'll ping @florian-sanders-cc for this one. :smile:  
Is it to specify the `lang` HTML attribute?

Reply:
Yes exactly ! :+1: